packages/legacy/runtime/test/getMesh.test.ts

```diff
@@ -213,4 +213,61 @@ describe('getMesh', () => {
       }
     `);
   });
+
+  it('logs the unexpected errors with stack traces', async () => {
+    const errorLogSpy = jest.spyOn(logger, 'error');
+    const mesh = await getMesh({
+      cache,
+      pubsub,
+      logger,
+      merger,
+      sources: [
+        createGraphQLSource({
+          suffix: 'Foo',
+          suffixRootTypeNames: false,
+          suffixFieldNames: true,
+          suffixResponses: true,
+        }),
+      ],
+      additionalTypeDefs: [
+        parse(/* GraphQL */ `
+          extend type Query {
+            throwMe: String
+          }
+        `),
+      ],
+      additionalResolvers: {
+        Query: {
+          throwMe: () => {
+            throw new Error('This is an error');
+          },
+        },
+      },
+    });
+
+    const result = await mesh.execute(
+      /* GraphQL */ `
+        query {
+          throwMe
+        }
+      `,
+      {},
+    );
+
+    expect(result).toMatchInlineSnapshot(`
+      {
+        "data": {
+          "throwMe": null,
+        },
+        "errors": [
+          [GraphQLError: This is an error],
```

Comment:
Shouldn't the stack be here too? I reckon it's useful info while developing.

Reply:
I added this;
https://github.com/ardatan/graphql-mesh/pull/6798/files#diff-51aaad92a28392a02317fec107fe1ddbaac85de1a7cdfeb944001b4093486b12R276
What do you think?